tools.py

```diff
@@ -1,24 +1,31 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
+import logging
 import os
 import shutil
 import subprocess
 import sys
-from exit_codes import ExitCode
+
 import output_unifier
-import logging
+from exit_codes import ExitCode
 
-ACCEPTED_EXTENSIONS = ["c", "cc", "cpp", "c++", "h", "hpp", "hh"]
+ACCEPTED_EXTENSIONS = ["c", "cc", "cpp", "c++", "h", "hpp", "hh", "rs"]
 
 _SUPPORTED_EXTENSIONS_CCCC_ = ["c", "cc", "cpp", "c++", "h", "hpp", "hh"]
-_SUPPORTED_EXTENSIONS_TOKEI_ = ACCEPTED_EXTENSIONS  # [""]   # Tokei supports a huge variety of languages
+_SUPPORTED_EXTENSIONS_TOKEI_ = (
+    ACCEPTED_EXTENSIONS  # [""]   # Tokei supports a huge variety of languages
+)
+_SUPPORTED_EXTENSIONS_RUST_CODE_ANALYSIS_ = ACCEPTED_EXTENSIONS
```

Comment:
Does this tool work on all of the supported extensions?

Reply:
Yes, it does